EventView-1/src/main/java/com/eventview/exceptions/EventTypeExceptionController.java

```diff
@@ -0,0 +1,28 @@
+package com.eventview.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.sql.Timestamp;
+
+@ControllerAdvice
+public class EventTypeExceptionController {
+    @ExceptionHandler(value = EventTypeNotFoundException.class)
```

Comment:
what is diff than EventExceptionController , why duplicate controllerAdvice for multiple exceptions